agh_graphs/utils.py

```diff
@@ -97,6 +97,31 @@ def add_break_in_segment(graph: Graph, segment: (str, str)) -> str:
     return v
 
 
+def add_break_in_segment(graph: Graph, segment: (str, str)) -> str:
```

Comment:
Nadal jest ten sam problem: ta funkcja już jest zdefiniowana w linii 74. Dodatkowo chcesz zmergować commit 92e15ebc9530649878edc7387fd7056d0edbe166, który jest podejrzany bo autorem jest @mickel8 . Jak dla mnie to tamten commit jest do wywalenia z historii i ta zmiana jest do revertu, wtedy będzie okej.